.eslintrc

```diff
@@ -488,6 +488,48 @@
         ]
       }
     },
+    {
```

Comment:
Why not add this config to the `no-restricted-imports` config above on line 480? Or at the very top on line 30?

Don't think we need a completely new override for this.

Reply:
Different excluded files and include files.